src/test/java/guitests/ContextMenuTests.java

```diff
@@ -81,6 +81,22 @@ public void test2() {
     }
 
     /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_twelveIssuesInListView_markIssue7andBelowReadUnread() {
+        scrollToTheEndOfThePanel();
+        for (int i = 0; i < 2; i++) {
+            markAndVerifyIssuesBelow(7, true);
+        }
+        for (int i = 0; i < 2; i++) {
+            markAndVerifyIssuesBelow(7, false);
+        }
+        markAndVerifyIssuesBelow(1, true);
+        markAndVerifyIssuesBelow(1, false);
```

Comment:
1 - to test for the corner case where there are no issues `below` but `on` is satisfied
7 - to account for the general case where we pick an issue in the middle or anywhere on the list and check whether the issues on/below have been marked

Empty list has been checked with context menu items disabled in the `Disabled` tests.